2200001/문제2.py

```diff
@@ -0,0 +1,18 @@
+# 파리 퇴치
+T = int(input())
+
+for t in range(1, T+1):
+    n, m = map(int, input().split())
+    array = [list(map(int, input().split())) for _ in range(n)]
+    
+    hits = []
+
+    for i in range(n-m+1):
+        for j in range(n-m+1):
+            fly = 0 
+            for k in range(m):
+                for l in range(m):
+                    fly += array[i+k][j+l]
+                hits.append(fly)
```

Comment:
이중 리스트 탐색 잘해주셨네요! 👍 
```suggestion
    for i in range(n-m+1):
        for j in range(n-m+1):
            fly = 0 
            for k in range(m):
                for l in range(m):
                    fly += array[i+k][j+l]
            hits.append(fly)
```
`hits` 리스트에 잡게 되는 파리의 수를 저장하려면, 앞의 여백을 지워주시는 것이 맞습니다! 현재 코드는 `hits` 리스트에 불필요한 데이터들이 많이 추가됩니다.